package.json

```diff
@@ -41,8 +45,7 @@
   "devDependencies": {
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.1.4",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1"
+    "eslint-plugin-react": "^7.8.2",
```

Comment:
Did you mean to add this as a dev dependency?

Reply:
No, I have removed this now